siemens/questa.mk

```diff
@@ -47,7 +53,8 @@ $(DEP_DIR)/%.questa.d: $(SIM_SUB_DONE) $(predependency_hook) | $(DEP_DIR) $(BLOG
 
 # targets: grep lines that have ':', remove cleans, sed drop last character
 # Extract all targets for sim:
-QUESTA_TARGETS := $(shell grep -ohe "^[a-z].*:" $(HDL_BUILD_PATH)/siemens/*.mk | grep -v clean | grep -v nuke | sed 's/:.*//')
+QUESTA_TARGETS := $(shell grep -ohe "^[a-z].*:" $(HDL_BUILD_PATH)/siemens/*.mk | grep -v autocheck | grep -v clean | grep -v nuke | sed 's/:.*//')
+AC_TARGETS := autocheck ac ac_gui
```

Comment:
This doesn't seem complete -- the `grep -v autocheck` will isolate `autocheck` as an `AC_TARGETS` value, but `ac` and `ac_gui` will still appear in `QUESTA_TARGETS`. 

It looks like the `grep -v autocheck` is not needed, and because `_TOP` is overriden with `$(TOP_AC)` below, all the `AC_TARGETS` properly set `$(TOP_AC)`.

I think you can remove ` | grep -v autocheck` and things will continue to work as is, and not have an unnecessary `grep` step.

Reply:
Or I could do `grep -vE "ac|auto"`

But yeah, overriding `_TOP :=` is easier